package.json

```diff
@@ -1,7 +1,38 @@
 {
+  "license": "MIT",
   "dependencies": {
     "form-data": "^2.1.2",
     "node-fetch": "^1.6.3",
-    "syncano-server": "^0.7.2-43"
+    "syncano-server": "beta"
+  },
+  "scripts": {
+    "test": "yarn && npm-run-all test:lint test:e2e:coverage",
+    "test:e2e-single": "cross-env NODE_ENV=test mocha -s 1800 --reporter spec --require babel-register --timeout 4000",
```

Comment:
This is useless. You can uses `describe.only` or `it.only`.

Reply:
Can you elaborate?